taworks/taform/templates/taform/application.html

```diff
@@ -234,13 +245,15 @@ <h2>Confirmation</h2>
 <p>To submit your application you must agree to the above terms.</p>
 
 <p>Please note you will not be able to make changes to your application once you submit.</p>
-<input name="confirm" type="submit" class="submitButton" id ="btn" value="Submit" disabled>
-</form>
 
-<div id="dialog">
-    <p>Are you sure you want to submit this application? Please ensure all information is correct.</p>
+<div id="app-confirm" title="Confirm Action">
+    <p class="appButtonPopupText">Are you sure you want to submit this application? Please ensure all information is correct.</p>
```

Comment:
Do we want to not put "please ensure all information is correct."?

we don't really give them a chance to ensure by showing what's been filled out / summary